src/queries/__tests__/a11yValue.test.tsx

```diff
@@ -46,15 +38,15 @@ test('getByA11yValue, queryByA11yValue, findByA11yValue', async () => {
   });
 
   expect(() => getByA11yValue({ min: 50 })).toThrow(
-    getNoInstancesFoundMessage('{"min":50}')
+    'Unable to find an element with min value: 50'
```

Comment:
Would you consider adding a test to see the error with both a min and max?

Reply:
I've added some more tests in this area.